### PR TITLE
refactor: Standardize MFA code and recovery code naming across code base

### PR DESCRIPTION
**Pull request by RicardoE105**
## Summary

This PR extracts the renaming from https://github.com/n8n-io/n8n/pull/12013

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Review by tomi — DISMISSED** [review]
Thanks for doing this 💟

File: packages/cli/src/controllers/auth.controller.ts [hunk 1]
```diff
@@ -75,16 +75,16 @@ export class AuthController {
 
 		if (user) {
 			if (user.mfaEnabled) {
-				if (!mfaToken && !mfaRecoveryCode) {
+				if (!mfaCode && !mfaRecoveryCode) {
 					throw new AuthError('MFA Error', 998);
 				}
 
-				const isMFATokenValid = await this.mfaService.validateMfa(
+				const isMFACodeOrMfaRecoveryCodeValid = await this.mfaService.validateMfa(
```
[the remark this answers is not in the document]
Comment (RicardoE105) [reply packages/cli/src/controllers/auth.controller.ts]: @tomi good catch!

**Review by tomi — APPROVED** [review]
🚀

**Comment by janober** [comment]
Got released with [`n8n@1.71.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.71.0)

**Pull request merged**